### DISCUSSION TITLE
how to update an array

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#16571] [Q&A] [answered]
I have an array I just want the array to be empty like `[1,2,3] => []` how do I update this?

**Comment by nurul3101** [comment, marked as answer]
Hello @rajamoulimallareddy 👋 

Considering that you have this model:
```prisma
model Question {
  id      Int      @id @default(autoincrement())
  answers String[]
}
```
You can override the values for answer field like this:
```ts
  await prisma.question.update({
    where: { id: 1 },
    data: {
      answers: {
        set: [],
      },
    },
  });
```
Here's a reference to [set](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#set) operator.

**Reply by rajamoulimallareddy** [reply]
and one more to remove the specific element in the array. what should I do?

**Reply by nurul3101** [reply]
You would have to fetch the scalar list from your database, modify it manually in your application code and overwrite the record in your database with an update operation like one above, there's no method to splice/remove items from a scalar list yet. We have a Feature Request for it here: [#7401](https://github.com/prisma/prisma/issues/7401), if you could leave a 👍 to it, then it would help our product team in prioritising it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
